index-providers/modeshape-elasticsearch-index-provider/src/test/java/org/modeshape/jcr/index/elasticsearch/client/EsClientTest.java

```diff
@@ -95,7 +95,7 @@ public void testStoreAndGetDocument() throws Exception {
         assertEquals(doc.get("field1"), doc1.get("field1"));
     }
 
-    @Test
+//    @Test
```

Comment:
@hchiorean , this is accidental commit, there are failed test so I left one for debugging the problem.